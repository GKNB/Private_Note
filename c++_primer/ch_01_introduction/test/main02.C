#include<iostream>
#include<string>

using namespace std;

template<typename T>
void readAndOutput()
{
  T tmp;
  while(cin >> tmp)
    cout << "This is tmp: " << tmp << endl;
}

void readAndOutputString()
{
  string str;
  while(getline(cin,str))
    cout << "This is str: " << str << endl;
}

int main()
{
  readAndOutput<string>();
}
