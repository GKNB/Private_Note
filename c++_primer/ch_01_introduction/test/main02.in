dsjfhfldk kdsj_9349o p uj 0qw`I 0= `=` 2`=-O-
DS FH8U19809009 083 9aop90 90dklf;k03]3=
dskjdk 0491-09 2l.; ./.2 

ds f  e okj]pk 
ds d l 
