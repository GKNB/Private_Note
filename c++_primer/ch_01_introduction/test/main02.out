This is tmp: dsjfhfldk
This is tmp: kdsj_9349o
This is tmp: p
This is tmp: uj
This is tmp: 0qw`I
This is tmp: 0=
This is tmp: `=`
This is tmp: 2`=-O-
This is tmp: DS
This is tmp: FH8U19809009
This is tmp: 083
This is tmp: 9aop90
This is tmp: 90dklf;k03]3=
This is tmp: dskjdk
This is tmp: 0491-09
This is tmp: 2l.;
This is tmp: ./.2
This is tmp: ds
This is tmp: f
This is tmp: e
This is tmp: okj]pk
This is tmp: ds
This is tmp: d
This is tmp: l
